playground/frontend/lib/modules/analytics/analytics_service.dart

```diff
@@ -16,119 +16,27 @@
  * limitations under the License.
  */
 
-import 'package:flutter/cupertino.dart';
-import 'package:playground/config.g.dart';
-import 'package:playground/modules/analytics/analytics_events.dart';
+import 'package:flutter/widgets.dart';
 import 'package:playground/modules/examples/models/example_model.dart';
 import 'package:playground/modules/sdk/models/sdk.dart';
 import 'package:provider/provider.dart';
-import 'package:usage/usage_html.dart';
-
-class AnalyticsService {
-  late final Analytics _analytics;
-
-  AnalyticsService() {
-    _analytics = AnalyticsHtml(kAnalyticsUA, 'beam', '1.0');
-  }
 
+abstract class AnalyticsService {
   static AnalyticsService get(BuildContext context) {
     return Provider.of<AnalyticsService>(context, listen: false);
   }
 
```

Comment:
The below looks like a lot of changes, but it is a mere renaming of the old `AnalyticsService.dart` to `GoogleAnalyticsService.dart`.